compiler/rustc_parse_format/src/lib.rs

```diff
@@ -986,6 +1009,19 @@ fn find_width_map_from_snippet(
     InputStringKind::Literal { width_mappings }
 }
 
+fn unescape_string(string: &str) -> Result<string::String, unescape::EscapeError> {
```

Comment:
```suggestion
fn unescape_string(string: &str) -> Option<string::String> {
```
The error is never used.

Maybe the whole function should also be inlined and removed, not sure.